pyproject.toml

```diff
@@ -13,21 +13,26 @@ pydantic = "^1.9.0"
 pandas = "^1.4.2"
 toolz = "^0.12.0"
 
-[tool.poetry.group.dev.dependencies]
-blacken-docs = "^1.12.1"
-Sphinx = "^4.5.0"
+
+[tool.poetry.group.doc.dependencies]
```

Comment:
Good point

Reply:
I don't see why we have to separate them